week3/cardgame/src/Modal.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+const Modal = ({ onClose }) => {
+  return (
+    <ModalWrapper>
+      <ModalContent>
+        <h2>축하합니다! 동물의 숲 카드게임 성공!</h2>
+        <button onClick={onClose}>닫기</button>
```

Comment:
p4: 여기 부분도 네이밍을 조금만 수정하면 너무 좋을 것 같아! 

```suggestion
const Modal = ({ onClickClose }) => {
  return (
    <ModalWrapper>
      <ModalContent>
        <h2>축하합니다! 동물의 숲 카드게임 성공!</h2>
        <button onClick={onClickClose}>닫기</button>
```
이렇게 작성하면 조금 더 클릭했을 때, 닫기 동작하는 함수인 걸 직관적으로 알 수 있을 것 같은데 어떨까?

Reply:
오 너무 좋은 것 같숨다! 반영했어요 감사합니다!! 👍🏻